capo/clean-os-resources.sh

```diff
@@ -1,55 +1,71 @@
 #!/bin/bash
 
-_prefix=${1:-"cluster-e2e"}
+FILTER=${1:-"no-existing-resources"}
+datadir="/tmp/collectedsources"
 
-echo "===Removing openstack servers==="
-servers=$(openstack server list -c Name -f value | grep ${_prefix})
-for server in ${servers};do 
-  echo "Removing server: ${server}"
-  openstack server delete "${server}";
-done
+remove_resource() {
```

Comment:
```suggestion
run_after_confirmation() {
```
Removing resources depends on the provided command, there is nothing inherently deletion-related in this function

Reply:
done